accelerator-core/build.gradle

```diff
@@ -68,8 +68,8 @@ dependencies {
         maven { url 'http://tokbox.bintray.com/maven' }
     }
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'androidx.appcompat:appcompat:1.1.0'
-    implementation 'com.opentok.android:opentok-solutions-logging:+'
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'com.opentok.android:opentok-solutions-logging:1.0.2'
```

Comment:
@igorwojda I think we could add 1.0.+ for logging.

Reply:
We should avoid dynamic dependencies as they are considered bad practice and can cause some problems like broken builds or no easy way to build lib from a given day (eg. state of the repo from 2 months ago).

Reply:
https://blog.danlew.net/2015/09/09/dont-use-dynamic-versions-for-your-dependencies/